### ISSUE TITLE
Want to show the floating panel based on views present in main view controller.

### ISSUE THREAD

**Issue by mallikarjuna0308** [scenee/FloatingPanel#480]
Hi @scenee, I am using your library and it is easy to use. Thank you for providing this wonderful library.
I have a requirement where I need to show the floating panel based on other views in the main view controller. So, Is there a way to set the y position of the floating panel relative to its superview?

**Comment by scenee** [comment]
Yes there is, if you present a panel as a child vc. You are able to add `FloatingPanelController.view` in any view and locate it, like `FloatingPanelController.addPanel(toParent:)`.
